### PR TITLE
COM-4728

### PR DESCRIPTION
**Pull request by german-martinez**
* Update pylint dependency version

### CODE REVIEW

File: fluster/decoder.py [hunk 1]
```diff
@@ -18,8 +18,8 @@
 from abc import ABC, abstractmethod
 from functools import lru_cache
 from shutil import which
-from typing import Type
-
+from typing import List, Type
+import typing
```
Comment (mdimopoulos) [inline fluster/decoder.py:22]:
```suggestion
from typing import List, Type, Any
```

File: fluster/decoder.py [hunk 2]
```diff
@@ -70,14 +70,11 @@ def __str__(self) -> str:
         return f"    {self.name}: {self.description}"
 
 
-DECODERS = []
+DECODERS: List[typing.Any] = []
```
Comment (rgonzalezfluendo) [inline fluster/decoder.py:73]:
```suggestion
DECODERS: List[Decoder] = []
```

File: .pylintrc [hunk 3]
```diff
@@ -339,8 +339,7 @@ max-module-lines=1000
 # separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
 # `trailing-comma` allows a space between comma and closing bracket: (a, ).
 # `empty-line` allows space-only lines.
-no-space-check=trailing-comma,
-               dict-separator
+no-space-check=no
```
[the remark this answers is not in the document]
Comment (german-martinez) [reply .pylintrc]: Restored. Pylint OK here

File: fluster/decoder.py (hunk 1 above)
Comment (german-martinez) [reply fluster/decoder.py]: Any typing no needed as we'll use [Decoder] instead

File: fluster/decoder.py (hunk 2 above)
Comment (german-martinez) [reply fluster/decoder.py]: Correct and updated

**Review by mdimopoulos — APPROVED** [review]
Tested with the latest changes and the list of decoders printed out appears to be correct

**Pull request merged**